spec/datadog/tracing/contrib/rack/http_route_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'datadog/tracing/contrib/support/spec_helper'
+
+require 'rack/test'
+
+require 'datadog'
+require 'datadog/tracing/contrib/rack/middlewares'
+
+RSpec.describe 'Rack testing for http.route' do
+  include Rack::Test::Methods
+
+  before do
+    Datadog.configure do |c|
+      c.tracing.instrument :rack
+    end
+  end
+
+  after do
+    Datadog.configuration.tracing[:rack].reset!
+  end
```

Comment:
Not sure that I have all the context, but maybe it makes sense to wrap the configuration change (which is shared) in the `ensure` section to be sure that tests are not leaking in case of the exception?

Sorry if it's not relevant, but just to be sure

```suggestion
  around do |example|
    Datadog.configure { |c| c.tracing.instrument :rack }
    example.run
  ensure
    Datadog.configuration.tracing[:rack].reset!
  end
```

Reply:
fixed [cae659f](https://github.com/DataDog/dd-trace-rb/pull/3849/commits/cae659f4f62b2b976356afe212ca899fcba3041c)